src/main/java/Deadline.java

```diff
@@ -0,0 +1,29 @@
+public class Deadline extends Task{
+    private String dateOfDeadline;
+
+    Deadline(String deadlineDescription, String dateOfDeadline,int deadlineTaskId){
```

Comment:
need a space between `)` and `{` here.
Tip: find out how to correct these spacing issues on the IDE automatically!